pkg/skaffold/deploy/kustomize.go

```diff
@@ -363,3 +368,13 @@ func buildCommandArgs(buildArgs []string, kustomizePath string) []string {
 
 	return args
 }
+
+func IsKustomizationPath(path string) bool {
```

Comment:
I'm wondering if we would want to add some logic to have at least one `resources:` section in the file to make it a kustomization file?

Reply:
yeah not a bad idea